cmd/pipeline/controller.go

```diff
@@ -416,6 +431,12 @@ func abortDuplicatedProwJobs(c reconciler, pj *prowjobv1.ProwJob) (*prowjobv1.Pr
 	pjs := getFilteredProwJobs(id, prowJobsToFilter)
 	// do not abort the newest prowjob
 	for i := 0; i < len(pjs)-1; i++ {
+		// If feature flag is enabled, skip aborting postsubmit jobs
+		if c.allowConcurrentPostsubmitJobs() && pjs[i].Spec.Type == prowjobv1.PostsubmitJob {
+			logrus.WithFields(pjutil.ProwJobFields(pjs[i])).Infof("Skipping abort for postsubmit job %s because AllowConcurrentPostsubmitJobs is enabled", pjs[i].Name)
+			continue
+		}
```

Comment:
I am not that familiar with pipeline controller, but this second skip logic is surprising to me; I would expect this to be handled by the conditional return on L421? If we get to this abort loop it means the input `pj` is not a postsubmit. I have not looked at the logic under `getFilteredProwJobs` and `createProwJobIdentifier` but do we expect to find jobs with the "same identifier" between job types, like a presubmit and a postsubmit with the same identifier?